src/utils/array.ts

```diff
@@ -0,0 +1,9 @@
+interface Array<T> {
+  swap(srcIndex: number, destIndex: number): Array<T>;
+}
+
+Array.prototype.swap = function swap<T>(this: T[], srcIndex: number, destIndex: number): Array<T> {
```

Comment:
This should work, but is there any particular need of creating polyfill and not just regular helper function?

Reply:
changed to regular function